v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Canal Hosted Install
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/'
+---
+
+## About Canal
+
+Canal allow users to easily deploy Calico and flannel networking
+together as a unified networking solution - combining Calico’s
+industry-leading network policy enforcement with the flannel overlay and
+non-overlay network connectivity options.
+
+Note that the Canal currently uses the Calico and flannel projects as is with
+no code modifications to either. Canal is simply a deployment pattern
+for installing and configuring the projects to work together seamlessly as a
+single network solution from the point of view of the user and orchestration
+system.
+
+
+## Installation
+
+When deploying your kubernetes cluster, please make sure it meets the
+[requirements](#requirements--limitations) at the bottom of this page.
+An easy way to create a cluster which meets these requirements is by following
+[the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+
+> **Note:** If you are upgrading from the Kubernetes 
+[1.6](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#for-kubernetes-16)
+or [1.5](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md#kubernetes-15)
+manifests from the (deprecated) Canal repo to the manifests here it is
+neccessary to [migrate your Calico configuration
+data](https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md)
+before upgrading. Otherwise, your cluster may lose connectivity after the 
+upgrade.                       
+
+### RBAC 
+  
+Before you install Canal, if your Kubernetes cluster has RBAC enabled, you'll
+need to create the following RBAC roles to allow API access to Calico.
+
+Apply the following manifest to create these necessary RBAC roles and bindings.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/rbac.yaml
+```
+> You can also [view the YAML in your browser.](https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/rbac.yaml)
+ 
+### Canal with Kubernetes Datastore
+
+The recommended Canal installation uses the Kubernetes API as the datastore,
+the manifest below installs Calico and Flannel configured to use the
+Kubernetes API.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/canal.yaml
+```
+> You can also [view the YAML in your browser.](https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.7/canal.yaml)
+
+
+### (or) Canal with Etcd Datastore
```

Comment:
"Etcd" is also always lowercased. Also, we are using sentence case in the headings...so "Etcd Datastore" --> "etcd datastore"